ADejbakhsh/GII
# Feature/graph exp

To test 
write GRAPH_EXP: 
And then start writing an expression, suggestion should show up

Another solution is to do <ctrl> + <space> on a line that start with GRAPH_EXP
